### PR TITLE
Ncgames 9

### PR DESCRIPTION
**Pull request by Greeny1248**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -13,153 +13,320 @@ beforeEach(() => {
...
-              expect(review.hasOwnProperty("votes")).toBe(true);
-              expect(review.hasOwnProperty("designer")).toBe(true);
-            });
-          });
+});
+test("GET /api/users : status:200 and response message", () => {
+  return request(app)
+  .get("/api/users")
+  .expect(200);
+});
```
Comment (NRMSMITH) [inline __tests__/app.test.js:37]: This test isn't needed as you'll expect 200 in the next test!

File: __tests__/app.test.js [hunk 2]
```diff
@@ -13,153 +13,320 @@ beforeEach(() => {
...
-        return request(app)
-          .get("/api/reviews")
-          .expect(200)
-          .then((res) => {
-            res.body.reviews.forEach((review) => {
-              expect(review.hasOwnProperty("comment_count")).toBe(true);
-            });
-          });
+      expect(res.body.users.length).toBeGreaterThan(1);
+    });
```
Comment (NRMSMITH) [inline __tests__/app.test.js:51]: I would put expect length assertion above the forEach, just for clarity.

**Review by NRMSMITH — APPROVED** [review]
Check my comments, apply them and then merge :D No need for another PR.

File: __tests__/app.test.js (hunk 1 above)
Comment (Greeny1248) [reply __tests__/app.test.js:37]: Thanks, i have a few of these throughout, i will delete them all :)

File: __tests__/app.test.js (hunk 2 above)
Comment (Greeny1248) [reply __tests__/app.test.js:51]: Will do thanks!

**Pull request merged**